content/en/monitors/types/metric.md

```diff
@@ -102,13 +102,15 @@ Any metric reporting to Datadog is available for monitors. Use the editor and th
 | Option           | Description                                                                                                                                                                   |
 |------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | average          | The series is averaged to produce a single value that is checked against the threshold. It adds the `avg()` function to your monitor query.                                   |
-| max              | If any single value in the generated series crosses the threshold, then an alert is triggered. It adds the max() function to your monitor query.                              |
-| min              | If all points in the evaluation window for your query cross the threshold, then an alert is triggered. It adds the min() function to your monitor query.                      |
+| max              | If any single value in the generated series crosses the threshold, then an alert is triggered. It adds the max() function to your monitor query.*                             |
+| min              | If all points in the evaluation window for your query cross the threshold, then an alert is triggered. It adds the min() function to your monitor query.*                     |
```

Comment:
I wonder if it could be even better to embed it into the description text itself, as the asterisk can be missed?

I see that it would make it longer, but maybe
```
max: If any single value (for "below" trigger) or all points in the evaluation window (for "above" triggers) in the generated series crosses the threshold, then an alert is triggered. It adds the max() function to your monitor query
```

Reply:
Thanks for bringing this to our attention! Another alternative is add "See Note for additional threshold behavior"
```suggestion
| min              | If all points in the evaluation window for your query cross the threshold, then an alert is triggered. It adds the min() function to your monitor query. See Notes for additional threshold behavior                    |
```